src/mrinufft/operators/interfaces/cufinufft.py

```diff
@@ -58,56 +58,55 @@ def __init__(
         **kwargs,
     ):
         self.shape = shape
-        self.samples = samples
         self.ndim = len(shape)
```

Comment:
We discussed it on board :) 
For posterity:

We dont want to have multiple states of samples (one in wrapper, one in raw_op and one inside raw_op). This can cause weird issues and make it hard to track